### ISSUE TITLE
meaning of data.FONTSIZESCALE

### ISSUE THREAD

**Issue by rettinghaus** [music-encoding/music-encoding#663]
I don't understand `data.FONTSIZESCALE`. I would have guessed, that "relative" means a percentage value. But an integer between 1 and 9? 
@pe-ro could you explain this, please?

**Comment by pe-ro** [comment]
Absolute values consist of a number and some kind of unit, e.g. "10 px".  The values here, however, are just numbers -- 1 is the smallest size, 9 is largest.  The actual rendition is dependent on/relative to the "base" or "normal" size, which is equivalent to the value "5".

Percentage values provide continuous values between 1 and 100. The integers 1 - 9 were chosen instead in order to give a smaller range of "steps" -- 4 on either side of a central value.  A smaller range of values is better for the items data.FONTSIZESCALE is applied to by default -- symbols attached at either end of a line -- because an arrow head, for example, at 60% wouldn't be noticeably different from one at 61%.

"Data binning (also called Discrete binning or bucketing) is a data pre-processing technique used to reduce the effects of minor observation errors. The original data values which fall into a given small interval, a bin, are replaced by a value representative of that interval, often the central value. It is a form of quantization." ([https://en.wikipedia.org/wiki/Data_binning](url))

data.FONTSIZESCALE is perhaps poorly named.  Maybe FONTSIZESCALED, FONTSIZEBINNED, FONTSIZEQUANTIZED, or FONTSIZESTEPPED would make more sense.

**Comment by rettinghaus** [comment]
Thanks. Yes, I think FONTSIZESTEPPED would be better. But we should note somewhere, that "5" means "normal size", and that rendition may differ from one application to another.

**Comment by lpugin** [comment]
I fully agree we can improve the description of the data type. I less sure about the name change. It does not bring much and I would leave it as it.
